Vungle/VungleRouter.m

```diff
@@ -444,6 +463,27 @@ - (void)clearWaitingList {
     [self.waitingListDic removeAllObjects];
 }
 
+- (void)requestBannerAdFailedWithError:(NSError *)error
+                           placementID:(NSString *)placementID
+                              delegate:(id<VungleRouterDelegate>)delegate {
+    if (error) {
+        NSLog(@"Vungle: Unable to load an ad for Placement ID :%@, Error %@", placementID, error);
```

Comment:
Should `NSLog` be `MPLogInfo` here?

Reply:
Got it, I will replace all NSLogs with MPLogInfos :)